### PR TITLE
Dynamic region content

### PR DESCRIPTION
**Pull request by smcmurtry**

### CODE REVIEW

File: src/screens/home/components/Tip.tsx [hunk 1]
```diff
@@ -16,16 +18,18 @@ export const Tip = () => {
...
         </Box>
       </Box>
       <Box paddingHorizontal="m" paddingTop="s">
         <ButtonSingleLine
-          text={i18n.translate(`Home.DiagnosedView.Tip.${region}.CTA`)}
+          text={regionalI18n.translate(`RegionContent.DiagnosedView.Active.${region}.Tip.CTA`)}
           variant="thinFlatNeutralGrey"
           externalLink
-          onPress={() => Linking.openURL(i18n.translate(`Home.DiagnosedView.Tip.${region}.URL`))}
+          onPress={() =>
```
Comment (timarney) [inline src/screens/home/components/Tip.tsx:30]: @smcmurtry looks like we missed this one

File: src/screens/nocode/views/ActiveParagraphView.tsx [hunk 2]
```diff
@@ -4,12 +4,16 @@ import {useI18n} from 'locale';
...
 
-export const NLView = () => {
+export const ActiveParagraphView = () => {
   const i18n = useI18n();
+  const regionalI18n = useRegionalI18n();
+  const {region} = useStorage();
   const autoFocusRef = useAccessibilityAutoFocus(true);
   const onClick = () =>
-    Linking.openURL(i18n.translate('DataUpload.NoCode.RegionCovered.NL.Link')).catch(error =>
+    Linking.openURL(i18n.translate(`DataUpload.NoCode.RegionCovered.${region}.Link`)).catch(error =>
```
Comment (timarney) [inline src/screens/nocode/views/ActiveParagraphView.tsx:16]: @smcmurtry ☝️

Comment (smcmurtry) [reply src/screens/nocode/views/ActiveParagraphView.tsx]: fixed!

File: src/screens/home/components/Tip.tsx (hunk 1 above)
Comment (smcmurtry) [reply src/screens/home/components/Tip.tsx:30]: It just shifted down a line: `Linking.openURL(regionalI18n.translate(`RegionContent.DiagnosedView.Active.${region}.Tip.URL`))`

**Comment by timarney** [comment]
Merging this, we'll finish up the remainder in separate PRs.

**Pull request merged**